java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/Application.java

```diff
@@ -49,56 +50,82 @@
 
   private static final Options CLI_OPTIONS = new Options();
   private static final String EXECUTABLE_NAME = "<BUILDER>";
+  public static final ImmutableMap<String, String> DEFAULT_JDK_MAPPINGS = ImmutableMap.of(
+      "*", "gcr.io/google-appengine/openjdk:8",
+      "openjdk8", "gcr.io/google-appengine/openjdk:8",
+      "openjdk9", "gcr.io/google-appengine/openjdk:9"
+  );
+  public static final ImmutableMap<String, String> DEFAULT_SERVER_MAPPINGS =
+      new ImmutableMap.Builder<String, String>()
+          .put("*|*", "gcr.io/google-appengine/jetty:9")
+          .put("openjdk8|*", "gcr.io/google-appengine/jetty:9")
+          .put("openjdk8|jetty9", "gcr.io/google-appengine/jetty:9")
+          .put("openjdk8|jetty", "gcr.io/google-appengine/jetty:9")
+          .put("openjdk8|tomcat8", "gcr.io/google-appengine/tomcat:8")
+          .put("openjdk8|tomcat", "gcr.io/google-appengine/tomcat:8")
+          .put("*|jetty9", "gcr.io/google-appengine/jetty:9")
+          .put("*|jetty", "gcr.io/google-appengine/jetty:latest")
+          .put("*|tomcat8", "gcr.io/google-appengine/tomcat:8")
+          .put("*|tomcat", "gcr.io/google-appengine/tomcat:latest").build();
+  public static final String DEFAULT_COMPAT_RUNTIME_IMAGE =
+      "gcr.io/google-appengine/jetty9-compat:latest";
+  public static final String DEFAULT_MAVEN_DOCKER_IMAGE =
+      "gcr.io/cloud-builders/java/mvn:3.5.0-jdk-8";
```

Comment:
`:latest` maybe?

Reply:
Is there a concern with using a fixed version? Now with the unit test if java.yaml gets updated we won't need to worry about this getting out of date.